src/verify.py

```diff
@@ -0,0 +1,18 @@
+def check(n: list, number: int) -> bool:
+    if number in n:
+        return True
+    return False
+
+def main(N: list, number: int) -> str:
+    flag = check(N, number)
+    
+    if flag is True: 
+        return f"The number {number} is present in the array"
+    else:
+        return f"The number {number} is not present in the array"
```

Comment:
you can delete the flag variable and put
if check...
   return ....
   
   moreover there is no need of the `else`, you can just write the second return below after the if like
```python
if check:
   return ...
   
return ...
```